classes/submission/reviewer/form/PKPReviewerReviewStep1Form.php

```diff
@@ -56,13 +56,12 @@ public function fetch($request, $template = null, $display = false)
         $context = $request->getContext();
 
         // Add submission parameters.
-        $reviewerSubmission = $this->getReviewerSubmission();
-        $templateMgr->assign('completedSteps', $reviewerSubmission->getStatus());
-        $templateMgr->assign('reviewerCompetingInterests', $reviewerSubmission->getCompetingInterests());
+        $reviewSubmission = $this->getReviewSubmission();
+        $reviewAssignment = $this->getReviewAssignment();
+        $templateMgr->assign('completedSteps', $reviewSubmission->getStatus());
```

Comment:
I was going to ask if `completedSteps` was correct, but it doesn't appear to be used in any template. I think it can probably be removed.

Reply:
When the reviewer check the review wizard on his interface (logged in as the reviewer), the interface "keeps track" of the wizard progress - during the first "version" of this fix the interface failed to keep track of the steps passed, and I tried to fix that. This was a part of the initial fix, but it's not needed any more.

Removed

Reply:
I see the completedSteps assignment in the main branch though (https://github.com/pkp/pkp-lib/blob/main/classes/submission/reviewer/form/PKPReviewerReviewStep1Form.php#L60) - On stable-3_3_0 also

Reply:
:+1: the variable is assigned but never used so it's good to remove it.

Reply:
removed here and on stable-3_3_0 also